{{cookiecutter.repo_name}}/setup.py

```diff
@@ -13,9 +13,9 @@
 with open("HISTORY.rst") as history_file:
     history = history_file.read()
 
-setup_requirements = ["pytest-runner"]  # type: List[str]
+setup_requirements = ["pytest-runner"]
```

Comment:
dont you need to also remove the typing import?

Reply:
Autoflake will do this whenever it is run on a project generated from this repository, but you're right I think it's neater to remove the import in the template itself